### PR TITLE
WIP - finetune elasticsearch

### PR DESCRIPTION
**Pull request by BriocheBerlin**

### CODE REVIEW

File: app/models/concerns/searchable.rb [hunk 1]
```diff
@@ -10,53 +10,89 @@ module Searchable
...
+                field: "cities.unmod"
               }
             }
           }
         })
     end
 
     def as_indexed_json(options={})
-      self.as_json(
+      as_json(
```
Comment (zaziemo) [inline app/models/concerns/searchable.rb]: why did you change class method into instance method?

File: app/views/profiles/index.html.erb [hunk 2]
```diff
@@ -9,25 +9,35 @@
...
+        </p>
+      <% else %>
+        <p class="text--color-blue font-size-24 text--center layout__item 1/1">
+          <%= t(:no_profile, link: link_to(t(:all_profiles, scope: 'search'), profiles_path), scope: 'search').html_safe %>
+        </p>
+      <% end %>
+
+
+    <div class="layout mb+ pt">
+      <% if @profiles.any? %>
```
Comment (tyranja) [inline app/views/profiles/index.html.erb]: Why do we have to check here again for @profiles? ( see line number 19

File: app/views/shared/_profile_search.html.erb [hunk 3]
```diff
@@ -7,8 +7,4 @@
     <%= button_tag((t(:search, scope: 'pages.home.search')), class: 'btn btn--small input-btn') %>
 
   <% end %>
-
```
Comment (tyranja) [inline app/views/shared/_profile_search.html.erb]: Does that form use the new elastic_search implementation?

File: app/views/profiles/index.html.erb (hunk 2 above)
Comment (tyranja) [inline app/views/profiles/index.html.erb]: Sorry, I got it now. Everything :+1:

**Comment by zaziemo** [comment]
I merge the branch despite the failing tests -> problem: elasticsearch index creation fails because of min_score. Thank you for your great work @BriocheBerlin <3
